.github/workflows/ci-workflow.yml

```diff
@@ -1,6 +1,7 @@
 name: Continuous Integration
 on:
   push:
```

Comment:
If we care about the double-jobs for PRs we could maybe do something like https://github.com/primer/view_components/blob/main/.github/workflows/ci.yml#L7-L9 to limit the push events to the main branch.

Reply:
Ended up doing this